tests/mc_loop_test/mc_loop_test.c

```diff
@@ -10,10 +10,16 @@
  * If you wish to obtain a commercial license, please contact Mellanox at support@mellanox.com.
  */
 
+/*
+ * To build the sample:
+ * gcc mc_loop_test.c -o mc_loop_test -g -Wall -DVMA_ZCOPY_ENABLED=1
+ *
```

Comment:
Could you add more details:
- how to compile using Berkeley API
- how to compile using vma_poll
Pay attention that probably -I and -L are needed to avoid issue when libvma.so installed at unusual location